dist/plot-schema.json

```diff
@@ -7109,6 +7109,93 @@
         "nonzero"
        ]
       },
+      "label": {
```

Comment:
This file shouldn't be updated except during the release process

https://github.com/plotly/plotly.js/blob/master/CONTRIBUTING.md#step-8-review--commit-potential-changes-made-to-testplot-schemajson